### ISSUE TITLE
AttributeError: module 'pandas' has no attribute 'NA'

### ISSUE THREAD

**Issue by sgrivers** [selik/xport#37]
Hello Mr. Selik,

I am trying to read a .XPT file into Python for a class assignment and I came across your library.  Using the sample code provided at https://pypi.org/project/xport/ , I am receiving the error:

AttributeError: module 'pandas' has no attribute 'NA'

Here is the code I used, modified from the website:

```
import xport
import xport.v56

with open('data/DXX_J.XPT','rb') as f:
    library = xport.v56.load(f)
```

​It is past my skill level to look into the source code and try to fix the error myself.

[xport_error.zip](https://github.com/selik/xport/files/4523979/xport_error.zip)

**Comment by sgrivers** [comment]
My professor's Docker container had a very old version of Pandas installed.  After upgrading Pandas, it now does indeed have the attribute 'NA'.

**Comment by selik** [comment]
@sgrivers Do you know what version of Pandas you had previously? I can mark a minimum version in `setup.cfg` so it'll warn you when you install.

**Comment by selik** [comment]
Checking the release notes, I should probably use `1.0`.

**Comment by selik** [comment]
Fixed with #38 .
